drivers/SmartThings/matter-thermostat/src/init.lua

```diff
@@ -174,12 +181,12 @@ local function get_device_type(driver, device)
 end
 
 local function do_configure(driver, device)
-  local heat_eps = device:get_endpoints(clusters.Thermostat.ID, {feature_bitmap = clusters.Thermostat.types.Feature.HEATING})
-  local cool_eps = device:get_endpoints(clusters.Thermostat.ID, {feature_bitmap = clusters.Thermostat.types.Feature.COOLING})
-  local auto_eps = device:get_endpoints(clusters.Thermostat.ID, {feature_bitmap = clusters.Thermostat.types.Feature.AUTOMODE})
+  local heat_eps = device:get_endpoints(clusters.Thermostat.ID, {feature_bitmap = clusters.Thermostat.types.ThermostatFeature.HEATING})
```

Comment:
This will work with both old and new (10) versions of the Lua Libs?

Reply:
Yes, the new lua libs feature aliases that will check for the this old name for the feature ('ThermostatFeature') and map it to the new name as of the 1.2 spec (just `Feature`)